structsect.go

```diff
@@ -32,8 +32,9 @@ type SectPr struct {
 
 // PgSz show the paper size
 type PgSz struct {
-	W int `xml:"w:w,attr"` // width of paper
-	H int `xml:"w:h,attr"` // high of paper
+	W      int    `xml:"w:w,attr"` // width of paper
+	H      int    `xml:"w:h,attr"` // high of paper
+	Orient string `xml:"w:orient,attr,omitempty"`
 }
 
 // UnmarshalXML ...
```

Comment:
You should also add the related `UnmarshalXML` segment of the newly added `Orient` field to ensure that it can be parsed normally.